src/um/taskschd.rs

```diff
@@ -0,0 +1,1158 @@
+        flags: LONG,
+        UserId: VARIANT,
+        password: VARIANT,
+        LogonType: TASK_LOGON_TYPE,
+        sddl: VARIANT,
+        ppTask: *mut *mut IRegisteredTask,
+    ) -> HRESULT,
+    fn GetSecurityDescriptor(
+        securityInformation: LONG,
+        pSddl: *mut BSTR,
+    ) -> HRESULT,
+    fn SetSecurityDescriptor(
+        sddl: BSTR,
+        flags: LONG,
+    ) -> HRESULT,
+}}
+RIDL!{#[uuid(0x9c86f320, 0xdee3, 0x4dd1, 0xb9, 0x72, 0xa3, 0x03, 0xf2, 0x6b, 0x06, 0x1e)]
+interface IRegisteredTask(IRegisteredTaskVtbl): IDispatch(IDispatchVtbl) {
+    fn get_Name(
+        pName: *mut BSTR,
+    ) -> HRESULT,
+    fn get_Path(
+        pPath: *mut BSTR,
+    ) -> HRESULT,
+    fn get_State(
+        pState: *mut TASK_STATE,
+    ) -> HRESULT,
+    fn get_Enabled(
+        pEnabled: *mut VARIANT_BOOL,
+    ) -> HRESULT,
+    fn put_Enabled(
+        pEnabled: VARIANT_BOOL,
+    ) -> HRESULT,
+    fn Run(
+        params: VARIANT,
+        ppRunningTask: *mut *mut IRunningTask,
+    ) -> HRESULT,
+    fn RunEx(
+        params: VARIANT,
+        flags: LONG,
+        sessionID: LONG,
+        user: BSTR,
+        ppRunningTask: *mut *mut IRunningTask,
+    ) -> HRESULT,
+    fn GetInstances(
+        flags: LONG,
+        ppRunningTasks: *mut *mut IRunningTaskCollection,
+    ) -> HRESULT,
+    fn get_LastRunTime(
+        pLastRunTime: *mut DATE,
+    ) -> HRESULT,
+    fn get_LastTaskResult(
+        pLastTaskResult: *mut LONG,
+    ) -> HRESULT,
+    fn get_NumberOfMissedRuns(
+        pNumberOfMissedRuns: *mut LONG,
+    ) -> HRESULT,
+    fn get_NextRunTime(
+        pNextRunTime: *mut DATE,
+    ) -> HRESULT,
+    fn get_Definition(
+        ppDefinition: *mut *mut ITaskDefinition,
+    ) -> HRESULT,
+    fn get_Xml(
+        pXml: *mut BSTR,
+    ) -> HRESULT,
+    fn GetSecurityDescriptor(
+        securityInformation: LONG,
+        pSddl: *mut BSTR,
+    ) -> HRESULT,
+    fn SetSecurityDescriptor(
+        sddl: BSTR,
+        flags: LONG,
+    ) -> HRESULT,
+    fn Stop(
+        flags: LONG,
+    ) -> HRESULT,
+    fn GetRunTimes(
+        pstStart: *const SYSTEMTIME,
+        pstEnd: *const SYSTEMTIME,
+        pCount: *mut DWORD,
+        pRunTimes: *mut *mut SYSTEMTIME,
+    ) -> HRESULT,
+}}
+ENUM!{enum TASK_STATE {
+    TASK_STATE_UNKNOWN = 0,
+    TASK_STATE_DISABLED = 1,
+    TASK_STATE_QUEUED = 2,
+    TASK_STATE_READY = 3,
+    TASK_STATE_RUNNING = 4,
+}}
+RIDL!{#[uuid(0x653758fb, 0x7b9a, 0x4f1e, 0xa4, 0x71, 0xbe, 0xeb, 0x8e, 0x9b, 0x83, 0x4e)]
+interface IRunningTask(IRunningTaskVtbl): IDispatch(IDispatchVtbl) {
+    fn get_Name(
+        pName: *mut BSTR,
+    ) -> HRESULT,
+    fn get_InstanceGuid(
+        pGuid: *mut BSTR,
+    ) -> HRESULT,
+    fn get_Path(
+        pPath: *mut BSTR,
+    ) -> HRESULT,
+    fn get_State(
+        pState: *mut TASK_STATE,
+    ) -> HRESULT,
+    fn get_CurrentAction(
+        pName: *mut BSTR,
+    ) -> HRESULT,
+    fn Stop(
+    ) -> HRESULT,
```

Comment:
If the function has no parameters, it should be a single line.

Reply:
done, thanks for the review!